src/workspaces/list/WorkspacesListModals.ts

```diff
@@ -29,7 +30,15 @@ export const WorkspacesListModals = (props: WorkspacesListModalsProps): ReactNod
       h(NewWorkspaceModal, {
         cloneWorkspace: userActions.cloningWorkspace,
         onDismiss: () => setUserActions({ cloningWorkspace: undefined }),
-        onSuccess: ({ namespace, name }) => goToPath('workspace-dashboard', { namespace, name }),
+        onSuccess: (ws) => {
+          if (userActions.cloningWorkspace && isGoogleWorkspace(userActions.cloningWorkspace)) {
```

Comment:
Please add unit test coverage for the difference in behavior (here and WorkspaceContainer).

Reply:
I'm working on that. There aren't any existing tests for the modal invocation, so the setup is kind of tricky.